.pipelines/singletenancy/cilium-nodesubnet/cilium-nodesubnet-e2e-step-template.yaml

```diff
@@ -58,7 +58,7 @@ steps:
         kubectl cluster-info
         kubectl get po -owide -A
         echo "install Cilium ${CILIUM_VERSION_TAG}"
-        export DIR=${CILIUM_VERSION_TAG%.*}
+        export DIR=$(echo ${CILIUM_VERSION_TAG#v} | cut -d. -f1,2)
```

Comment:
Curious to know the behavior if the version does not have `v` in the version tag ? 
Context: Earlier OSS version did not have a `v` in the version, but our private build image have `v`. In the future, if we want to get rid of private build images so we might not have `v` in the tag.(not sure though)

Reply:
The first part of that operation is "remove `v` if present". So we are fine in both cases. In fact, when we merged #3089, we kept the pipeline variables unchanged (so no `v`). Then separately updated the variables.